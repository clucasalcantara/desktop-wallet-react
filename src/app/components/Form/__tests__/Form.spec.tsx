import React from "react";
import { render } from "@testing-library/react";

import { Form, FormError } from "../";

describe("Form", () => {
	it("should render", () => {
		const { container, asFragment } = render(<Form />);

		expect(container).toBeTruthy();
		expect(asFragment()).toMatchSnapshot();
	});

	it("should render the error", () => {
		const { container, asFragment } = render(<FormError />);

		expect(container).toBeTruthy();
		expect(asFragment()).toMatchSnapshot();
	});
});
