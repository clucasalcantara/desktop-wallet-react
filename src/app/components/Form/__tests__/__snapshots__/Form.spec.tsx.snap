// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should render 1`] = `
<DocumentFragment>
  <form />
</DocumentFragment>
`;

exports[`Form should render the error 1`] = `
<DocumentFragment>
  <span
    class="inline-block mt-1 text-theme-danger-500"
    data-testid="form__input--error"
  />
</DocumentFragment>
`;
